verte182/Docker_Patrik/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# 
+FROM alpine:latest
+# 
+WORKDIR /home/ubuntu/Desktop/Docker_Patrik 
+# 
+COPY . /home/ubuntu/Desktop/Docker_Patrik
+# 
+RUN apk add --no-cache bash 
+# 
+EXPOSE 8080 
+# 
+ENV TZ=UTC 
+# 
+RUN ash /home/ubuntu/Desktop/Docker_Patrik/show_processes.sh 
+# 
+ENTRYPOINT ["ash", "-c"] 
+# 
+CMD ["echo 'Aktuální datum a čas: '; date"]
```

Comment:
vetsinou to bude tak, ze entrypoint bude ten bash script 
```
ENTRYPOINT ["./show_processes.sh "] 
```
script by se mel spustit sam, pokud tam das spravny shebang a udeals chmod +x